doc/api/HostDB.md

```diff
@@ -166,6 +167,130 @@ any particular order, and the order may change in subsequent calls.
+    // single download request.
+    "maxdownloadbatchsize": 17825792,
+
+    // Maximum duration in blocks that a host will allow for a file contract.
+    // The host commits to keeping files for the full duration under the
+    // threat of facing a large penalty for losing or dropping data before
+    // the duration is complete. The storage proof window of an incoming file
+    // contract must end before the current height + maxduration.
+    //
+    // There is a block approximately every 10 minutes.
+    // e.g. 1 day = 144 blocks
+    "maxduration": 25920,
+
+    // Maximum size in bytes of a single batch of file contract
+    // revisions. Larger batch sizes allow for higher throughput as there is
+    // significant communication overhead associated with performing a batch
+    // upload.
+    "maxrevisebatchsize": 17825792,
+
+    // Remote address of the host. It can be an IPv4, IPv6, or hostname,
+    // along with the port. IPv6 addresses are enclosed in square brackets.
+    "netaddress": "123.456.789.0:9982",
+
+    // Unused storage capacity the host claims it has, in bytes.
+    "remainingstorage": 35000000000,
+
+    // Smallest amount of data in bytes that can be uploaded or downloaded to
+    // or from the host.
+    "sectorsize": 4194304,
+
+    // Total amount of storage capacity the host claims it has, in bytes.
+    "totalstorage": 35000000000,
+
+    // Address at which the host can be paid when forming file contracts.
+    "unlockhash": "0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef0123456789ab",
+
+    // A storage proof window is the number of blocks that the host has to
+    // get a storage proof onto the blockchain. The window size is the
+    // minimum size of window that the host will accept in a file contract.
+    "windowsize": 144,
+
+    // Public key used to identify and verify hosts.
+    "publickey": {
+      // Algorithm used for signing and verification. Typically "ed25519".
+      "algorithm": "ed25519",
+
+      // Key used to verify signed host messages.
+      "key": "RW50cm9weSBpc24ndCB3aGF0IGl0IHVzZWQgdG8gYmU="
+    },
+
+	// The string representation of the full public key, used when calling
+	// /hostdb/hosts.
+	"publickeystring": "ed25519:1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef"
+  },
+
+  // A set of scores as determined by the renter. Generally, the host's final
+  // final score is all of the values multiplied together. Modified renters may
+  // have additional criteria that they use to judge a host, or may ignore
+  // certin criteia. In general, these fields should only be used as a loose
+  // guide for the score of a host, as every renter sees the world differently
+  // and uses different metrics to evaluate hosts.
+  "scorebreakdown": {
+    // The multiplier that gets applied to the host based on how long it has
+	// been a host. Older hosts typically have a lower penalty.
+    "ageadjustment":              0.1234,
```

Comment:
I just didn't want it to be interpreted as a canonical value. While right now you can get a good idea for host score by multiplying all the floats together, there's no api promise that it will continue to work that way.

I would make the promise, but things will get messy when we add host buckets. (geo stuff, etc.)